### PR TITLE
f-vue-icons@v1.8.0 – Eyeglass icon updated

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Changed

- Eyeglass icon updated.

### CODE REVIEW

File: packages/f-vue-icons/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.8.0
+------------------------------
+*October 29, 2020*
```
Comment (dandel10n) [inline packages/f-vue-icons/CHANGELOG.md]: October 30, 2020?

**Review by simonsandersje — DISMISSED** [review]

**Review by dandel10n — COMMENTED** [review]
PR name should start with  f-vue-icons@v1.8.0 and the date, other than that lgtm

File: packages/f-vue-icons/CHANGELOG.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/f-vue-icons/CHANGELOG.md]: 🤦

Comment (kevinrodrigues) [reply packages/f-vue-icons/CHANGELOG.md]: Fixed.

**Review by ashleynolan — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Review by dandel10n — APPROVED** [review]

**Pull request merged**